### PR TITLE
Create migrations for connected groups

### PR DESCRIPTION
**Pull request by zak39**

### CODE REVIEW

File: lib/Service/Group/GroupFormatter.php
Comment (smarinier) [inline lib/Service/Group/GroupFormatter.php:1]: I'm not fan of the is_ldap.
1) We had a discussion about it, and we agreed on the fact there will be "non WorkSpace groups". So added_groups basicaly (not especially LDAP)
2) The LDAP type detection is not very reliable.

So could we remove this principle ?

File: src/SelectUsers.vue
Comment (smarinier) [inline src/SelectUsers.vue:1]: Are this const values coming from somewhere in NextCloud ?

File: appinfo/routes.php
[the remark this answers is not in the document]
Comment (zak39) [reply appinfo/routes.php:1]: Hmmm... Good questions ! Do we need to get connected groups from a workspace or a group for this feature? :thinking: 

We could remove theses routes and functions in the controller and we could revisit it if necessary.

File: lib/Db/ConnectedGroupMapper.php
[the remark this answers is not in the document]
Comment (zak39) [reply lib/Db/ConnectedGroupMapper.php:1]: Oooh !! Good point ! I will fix it ! Thanks :wink:

File: lib/Service/Group/GroupFormatter.php
Comment (zak39) [reply lib/Service/Group/GroupFormatter.php:1]: Yes, of course, I created this pull request to "save" my work.
I will remove this key.

File: lib/Controller/GroupController.php
[the remark this answers is not in the document]
Comment (zak39) [reply lib/Controller/GroupController.php:1]: You mean in all the app?

We could do this later, in another PR if necessary :wink:

File: src/SelectUsers.vue
Comment (zak39) [reply src/SelectUsers.vue:1]: What values?

**Pull request merged**